timed/mixins.py

```diff
@@ -0,0 +1,72 @@
+from rest_framework_json_api.relations import ResourceRelatedField
+
+
+class AggregateQuerysetMixin(object):
+    """
+    Add support for aggregate queryset in view.
+
+    Wrap queryst dicts into aggregate object to support renderer
+    which expect attributes.
+    It additional prefetches related instances represented as id in
+    aggregate.
+
+    In aggregates only an id of a related field is part of the object.
```

Comment:
"In" => "It"?

Reply:
Well spotted... :) I rewrote the doc string a little bit to make it more comprehensible.